libnetdata/inlined.h

```diff
@@ -31,6 +31,29 @@ static inline uint32_t simple_hash(const char *name) {
     return hval;
 }
 
+/**
+ * Simple N hash
+ *
+ * Calculate the hash for name that has the length len.
+ *
+ * @param name the input to the hash
+ * @param len the length of the input
+ *
+ * @return It returns the hash calculated.
+ */
+static inline uint32_t simple_nhash(const char *name,size_t len) {
```

Comment:
Code style, you should add a space so that it is: `const char *name, size_t len`

Reply:
All right, I am fixing this with the last commit!